crypto/sign_test_utils.go

```diff
@@ -153,6 +153,10 @@ func testKeyGenSeed(t *testing.T, salg SigningAlgorithm, minLen int, maxLen int)
 	})
 }
 
+var BLS12381Order = []byte{0x73, 0xED, 0xA7, 0x53, 0x29, 0x9D, 0x7D, 0x48, 0x33, 0x39,
```

Comment:
```suggestion
const BLS12381Order = []byte{0x73, 0xED, 0xA7, 0x53, 0x29, 0x9D, 0x7D, 0x48, 0x33, 0x39,
```

Reply:
I don't think it's possible to make declare an array as a constant, an array in Go cannot be immutable.